charts/cromiam/templates/ingress/backendconfig.yaml

```diff
@@ -1,22 +1,4 @@
-{{ if .Values.ingress.enabled -}}
-apiVersion: cloud.google.com/v1
-kind: BackendConfig
-metadata:
-  name: {{ .Chart.Name }}-ingress-backendconfig
-  labels:
-{{ include "cromiam.labels" . | indent 4 }}
-spec:
-  timeoutSec: {{ .Values.ingress.timeoutSec }}
-  healthCheck:
-    checkIntervalSec: 5
-    timeoutSec: 5
-    healthyThreshold: 2
-    unhealthyThreshold: 2
-    type: HTTPS
-    port: 443
-    requestPath: /engine/latest/version
-{{- if .Values.ingress.securityPolicy }}
-  securityPolicy:
-    name: {{ .Values.ingress.securityPolicy }}
```

Comment:
Cromiam is a private and service and is behind a security policy firewall. I just want to make sure you've accounted for that case in the ingress library?

Reply:
Oooh! Good question and I have [here](https://github.com/broadinstitute/terra-helm/blob/d667b303a4589fffaa2b628bdcffb350178ada45/charts/ingresslib/templates/_backendconfig.yaml#L19)